ios/FluentUI/TwoLineTitleView/TwoLineTitleViewTokenSet.swift

```diff
@@ -86,16 +86,10 @@ extension TwoLineTitleViewTokenSet {
         }
     }
 
-    static let leadingImageSize = GlobalTokens.icon(.size160)
-    static let leadingImageMargin = GlobalTokens.spacing(.size40)
-    static let leadingImageTotalPadding: CGFloat = leadingImageSize + leadingImageMargin
+    static let minimumTouchSize = CGSize(width: 44, height: 44)
```

Comment:
nit. can we share this value with EastyTapButton definition?

Reply:
or actually.. are you trying to delete EasyTapButton all togeher?

Reply:
at the very least the easybutton now needs to have deprecation warning

Reply:
I'm not looking to delete `EasyTapButton`. Either way, having a single source of truth for the minimum touch size isn't a bad idea.